src/_gettsim/shared.py

```diff
@@ -98,6 +85,8 @@ def inner(func: Callable) -> Callable:
         func.__info__["dates_active_start"] = start_date
```

Comment:
```suggestion
        func.__info__["start_date"] = start_date
```
I think we can simplify a lot here now! The keys in the `__info__` dict should generally match the names of inputs to the decorator, I think. Unless heavily processed, but parsing dates seems below that bar to me.